src/Lykke.Service.MarketMakerReports.AzureRepositories/InventorySnapshotRepository.cs

```diff
@@ -13,10 +14,66 @@ namespace Lykke.Service.MarketMakerReports.AzureRepositories
     public class InventorySnapshotRepository : IInventorySnapshotRepository
     {
         private readonly INoSQLTableStorage<InventorySnapshotEntity> _storage;
+        
+        /// <summary>
+        /// IndexStorage is used to quickly get the last Snapshot for any date.
```

Comment:
Can we add another indexes for every Period (day, hour, etc) to make faster selections?
Or may be store additional records with RowKey = Period ?
For example when we selecting snapshots for a month or two, will it be slow?

Reply:
Nice suggestions, let's create a separate task for that in case we encounter a performance issues.